dtmi/innodisk/innoage/innoagectrl-1.json

```diff
@@ -0,0 +1,72 @@
+{
+    "@context": [
+        "dtmi:iotcentral:context;2",
+        "dtmi:dtdl:context;2"
+    ],
+    "@id": "dtmi:innodisk:innoage:innoagectrl;1",
+    "@type": "Interface",
+    "contents": [
+        {
+            "@type": "Property",
+            "displayName": {
+                "en": "SN"
+            },
+            "name": "innoageSN",
+            "schema": "string",
+            "writable": false
```

Comment:
writable is false by default, you can omit it here

Reply:
Does this means I can remove the key "writeable"?